### ISSUE TITLE
[BUG REPORT] Unresponsive UI, Missing TensorboardX library, wrong torchvision version installed.

### ISSUE THREAD

**Issue by d8ahazard** [gitmylo/audio-webui#214]
**Describe the bug**
So, there seem to be a handful of issues when installing on Linux. Just cloning the repo and running python ./main.py, or install.sh, or run.sh all gives the same results...

1. After installing requirements, I get a warning that tensorboardX is not installed. This has happened for a while now, and I've just been manually installing it.

2. I get an additional error regarding the version of TorchVision:

    raise ValueError(
ValueError: Could not find the operator torchvision::nms. Please make sure you have already registered the opera                                                              tor and (if registered from C++) loaded it via torch.ops.load_library.

Reinstalling torch/torchvision/torchaudio solves this:

pip install torch==2.1.0 torchvision==0.16.0 torchaudio==2.1.0 xformers --index-url https://download.pytorch.org/whl/cu118

3. Even after doing the above fixes, (or maybe because of them), once I load the webUI, I can't interact with anything. Selecting "suno/bark", for example, will just say "processing" forever, and nothing is shown in the logs...

**To Reproduce**
1. Install on linux manually as described in the readme...
2. Address errors.
3. Try to use the UI.

**Expected behavior**
The app works.

**Screenshots**
<img width="766" alt="image" src="https://github.com/gitmylo/audio-webui/assets/1633844/bb354027-3161-4fec-9257-63c1d5e5b433">

**Additional context**
N/A

**Comment by gitmylo** [comment]
Yeah, probably related to the update, I'll make sure the versions are hard set.
The tensorboard message can be ignored however, it isn't used and doesn't cause any errors.

**Comment by d8ahazard** [comment]
@gitmylo - Sorry, I updated the original post after you responded.

I updated my version and reinstalled vanilla, no tensorboardX, no updating torch, etc. All those errors are gone. 

However, the UI is still unresponsive when I try to do anything, and nothing in the logs, even with --verbose enabled.

Looking at network logs in my browser, I can see that the last message being sent is the request to load the model, then the server just...does nothing?

<img width="735" alt="image" src="https://github.com/gitmylo/audio-webui/assets/1633844/42b5a16a-1775-4fab-a83f-e163369ea2c3">
